Signal/src/ViewControllers/BlockListUIUtils.m

```diff
@@ -27,10 +29,12 @@ + (void)showBlockContactActionSheet:(Contact *)contact
     if (phoneNumbers.count < 1) {
         DDLogError(@"%@ Contact has no phone numbers", self.tag);
         OWSAssert(0);
-        [self showBlockFailedAlert:fromViewController];
-        if (completionBlock) {
-            completionBlock(NO);
-        }
+        [self showBlockFailedAlert:fromViewController
```

Comment:
Not calling the completion block until after the confirmation/failure alerts have been dismissed, allows us to present alerts/vcs in the completion handler.